include/hostio.h

```diff
@@ -36,7 +36,7 @@ VM_HOOK(account_codehash) void account_codehash(const uint8_t * address, uint8_t
  * that of the EVM. This means that, under the hood, this hostio is accessing the 32-byte
  * value stored in the EVM state trie at offset `key`, which will be `0` when not previously
  * set. The semantics, then, are equivalent to that of the EVM's [`SLOAD`] opcode.
- * 
```

Comment:
Let's re-add these, since they're the way the blocks are currently formatted

Reply:
:+1: